game-core/src/main/java/games/strategy/engine/framework/startup/ui/panels/main/MainPanel.java

```diff
@@ -58,18 +59,15 @@
       .preferredHeight(62)
       .build();
   private SetupPanel gameSetupPanel;
-  private boolean isChatShowing;
-  private final Supplier<Optional<ChatModel>> chatPanelSupplier;
 
   /**
    * MainPanel is the full contents of the 'mainFrame'. This panel represents the welcome screen and subsequent screens.
    */
   MainPanel(
       final GameSelectorPanel gameSelectorPanel,
       final Consumer<MainPanel> launchAction,
-      final Supplier<Optional<ChatModel>> chatPanelSupplier,
+      @Nullable final ChatModel chatModel,
```

Comment:
It looks like we could keep things simple here, when the setup panel is updated, it is passed to us and has attached to it a chat model. 

I'm not sure if we actually needed the supplier to begin with, I think the new setup panel will have the same chat model anyways.. Could be room here for more simplifications, for now this seems like a solid simplification.